src/main/java/seedu/address/logic/commands/ClearCommand.java

```diff
@@ -11,6 +11,9 @@
 public class ClearCommand extends Command {
 
     public static final String COMMAND_WORD = "clear";
+
+    public static final String USER_TIP = "You may clear the sample data with: " + COMMAND_WORD;
```

Comment:
this may sound ok for new users but for long time users, it may be a bit strange to see this. Maybe can rephrase to `"You may clear all data with: "`. wdyt?